### PR TITLE
Merge similar translation strings and avoid using HTML tags in translation strings

### PR DESCRIPTION
**Pull request by ramiy**
Based on my trac tickets:
https://glotpress.trac.wordpress.org/ticket/504
https://glotpress.trac.wordpress.org/ticket/505

See the screenshot (and attached patch).
We are merging similar strings with different variations and avoid using HTML `<code>` tags in translation strings.

### CODE REVIEW

File: gp-templates/helper-functions.php [hunk 1]
```diff
@@ -87,7 +87,7 @@ function textareas( $entry, $permissions, $index = 0 ) {
 			$warning = each( $referenceable );
 			?>
 			<div class="warning secondary">
-				<?php printf( __( '<strong>Warning:</strong> %s', 'glotpress' ), esc_html( $warning['value'] ) ); ?>
+				<strong><?php _e( 'Warning:', 'glotpress' ); ?></strong> <?php esc_html_e( $warning['value'] ); ?>
```
Comment (ocean90) [inline gp-templates/helper-functions.php]: `esc_html_e()` is used incorrect, should be `echo esc_html()`.

File: gp-includes/template.php [hunk 2]
```diff
@@ -433,7 +433,12 @@ function gp_project_options_form( $project ) {
 					<dt><label for="source-url-template">' . __( 'Source file URL', 'glotpress' ) . '</label></dt>
 					<dd>
 						<input type="text" value="' . esc_html( $project->source_url_template() ) . '" name="source-url-template" id="source-url-template" />
-						<small>' . __( 'URL to a source file in the project. You can use <code>%file%</code> and <code>%line%</code>. Ex. <code>https://trac.example.org/browser/%file%#L%line%</code>', 'glotpress' ) .'</small>
+						<small>' . sprintf(
+							__( 'Public URL to a source file in the project. You can use %1$s and %2$s. Ex. %3$s', 'glotpress' ),
```
Comment (ocean90) [inline gp-includes/template.php:437]: Don't forget to add translator comments.

File: gp-includes/cli/import-originals.php [hunk 3]
```diff
@@ -34,7 +34,7 @@ public function __invoke( $args, $assoc_args ) {
 
 		$translations = $format->read_originals_from_file( $args[1], $project );
 		if ( !$translations ) {
-			WP_CLI::error( __( "Couldn't load translations from file!", 'glotpress' ) );
+			WP_CLI::error( __( 'Couldn&#8217;t load translations from file!', 'glotpress' ) );
```
Comment (ocean90) [inline gp-includes/cli/import-originals.php]: `&#8217;` can't be used because it runs in CLI context.

File: gp-includes/template.php (hunk 2 above)
Comment (ramiy) [inline gp-includes/template.php:437]: Translation comments in a separate pull request. I will add comments to all the strings.

File: gp-templates/helper-functions.php (hunk 1 above)
Comment (ramiy) [inline gp-templates/helper-functions.php]: Fixed.

File: gp-includes/cli/import-originals.php (hunk 3 above)
Comment (ramiy) [inline gp-includes/cli/import-originals.php]: Reverted.

**Comment by ramiy** [comment]
@ocean90, thanks for the feedback.

**Comment by toolstack** [comment]
Hi @ramiy , as part of our move to the new repo this PR was closed when we merged back to "master" from "wordpress-dbi-plugin". If you still want to get it merged, just create a new PR to the "master" branch of the GlotPress/GlotPress-WP repo.

Thanks and sorry of the inconvenience.

**Comment by ramiy** [comment]
@toolstack: https://github.com/GlotPress/GlotPress-WP/pull/217

**Comment by toolstack** [comment]
Thank @ramiy.

**Pull request closed without merging**